src/transforms/remainder.jl

```diff
@@ -10,18 +10,78 @@ and returns a new table with an additional column containing the
 remainder value `xₙ₊₁ = total .- (x₁ + x₂ + ⋯ + xₙ)` If the `total`
 value is not specified, then default to the maximum sum across rows.
 """
-struct Remainder <: Transform end
+struct Remainder <: Transform
+  total::Union{Float64,Nothing}
+
+  function Remainder(total=nothing)
+    new(total)
+  end
+end
 
 isrevertible(::Type{Remainder}) = true
 
-function apply(::Remainder, table)
-  # TODO
+assertions(::Type{Remainder}) = [TT.assert_continuous]
+
+function apply(transform::Remainder, table)
+  # basic checks
+  for assertion in assertions(transform)
+    assertion(table)
+  end
+
+  # table as matrix and get the sum acros dims 2
+  X = Tables.matrix(table)
+  Σ = sum(X, dims=2)
```

Comment:
Please use S for sum. We already use \Sigma for covariances in other parts of the code if I am not mistaken.